jest.config.ts

```diff
@@ -28,4 +28,5 @@ export default {
     },
   },
   globalTeardown: './tearDown.cjs',
+  moduleFileExtensions: ['js', 'mjs', 'cjs', 'jsx', 'ts', 'tsx', 'json', 'node', 'd.ts'],
```

Comment:
This is necessary so jest can retrieve the `Type` enum from `appsync.d.ts` while running tests